smartmin/views.py

```diff
@@ -97,34 +95,7 @@ def has_permission(self, request, *args, **kwargs):
         if not getattr(self, 'permission', None):
             return True
         else:
-            # first check our anonymous permissions
-            real_anon = get_anonymous_user()
-            has_perm = real_anon.has_perm(self.permission)
```

Comment:
So smartmin no longer has a story for having views with permissions set but which are available to all? You just need to have no permission set?